### DISCUSSION TITLE
Cannot remove full nodes via Chia Gui

### DISCUSSION THREAD

**Discussion by hansgfixed** [Chia-Network/chia-blockchain#12977] [Support]
Since the first release of the Chia Gui it's unfortunately still not possible to remove full node connections via Gui. Is there any workaround available?

Thanks in advance!

**Comment by hansgfixed** [comment]
Any news on this? As this error persists since the very first release there must be someone being aware of this.

**Reply by OverActiveBladderSystem** [reply]
Prior to 1.5.1 in the Windows GUI you were able to disconnect a "full node connection" by clicking on the trash can located to the far right of it, now that the trash can has been removed in this latest version (hopefully they put it back in), you can follow the steps below for Windows...

first you must open a command prompt (click the windows start button, scroll down to the windows system folder, and click on the "command prompt" program within it)

second you will use the command below to switch to the directory Chia commands are located within (no need to change username, as it will use your current username, and app-* will use your current version installed)

`cd c:\Users\%USERNAME%\AppData\Local\chia-blockchain\app-*\resources\app.asar.unpacked\daemon\`

third you must know the "Node ID" of the node you wish to disconnect, you can view this in the GUI under the "Full Node" tab, if you scroll down to the "Full Node Connections" on that screen you will see their ID's listed that begin with `0x`

For example, if you see the Node ID you want to disconnect is `0x24fa1cfdbd3c0cb65add...` then you would type the command below to disconnect them (you only use the 8 characters after 0x)

`chia show -r 24fa1cfd` (every node will have a unique ID and you must change this for each node)

Now you should see that node removed from the GUI because you just disconnected it, you may now close the command prompt window or disconnect another peer if you wish to.

**Reply by hansgfixed** [reply]
Thanks but I get the error `Invalid NodeID. Do not include '.'`. But the whole string doesn't contain a dot.

**Reply by OverActiveBladderSystem** [reply]
That means you are using too few or too many characters for the NodeID, reference my example again, you ONLY use the eight alphanumerical characters AFTER `0x` meaning if the NodeID is `0x24fa1cfdbd3c0cb65add...` then you enter `24fa1cfd` only, no more, no less, and do not use `0x` at all, only what comes after it.

**Reply by hansgfixed** [reply]
@OverActiveBladderSystem 
Thanks!!

**Reply by HVNC1** [reply]
хотелось бы не танцевать с бубнами а вернуть как было
